src/main/resources/messages_en.properties

```diff
@@ -1,2 +1,3 @@
 file.not.empty=Please select a file to upload
-file.only.image=You can upload only jpeg images
\ No newline at end of file
+file.only.image=You can upload only jpeg images
+hex.invalid=Please use a valid hex parameter for color
```

Comment:
Do you still need this message? If not, please remove it

Reply:
removed